hazelcast/test/src/HazelcastTests4.cpp

```diff
@@ -1140,6 +1140,26 @@ namespace hazelcast {
 
                 }
 
+                TEST_F(RawPointerMapTest, testTryLockTtl) {
+
+                    ASSERT_TRUE(imap->tryLock("key1", 2 * 1000, 3 * 1000));
```

Comment:
3 secs lease time will cause this test to last at least 3secs. we may decrease it for this positive test scenario to keep test case time shorter.

Reply:
Decreased to 1.5 seconds